src/sass/components/_links.scss

```diff
@@ -1,7 +1,6 @@
 // Copyright (C) 2018 The Trustees of Indiana University
 // SPDX-License-Identifier: BSD-3-Clause
 
-a,
```

Comment:
I would say that we might want to leave this for now until we figure how how we're going to approach scoping the css for general HTML elements. I believe this is the only place where the link color is defined right now, so removing the `a` selector from this list right now, might have some unintended consequences that are maybe outside the scope of this PR.

Reply:
@levimcg good call! I've put the `a` back into the `_links.scss` file.